perf/benchmark/runner/generate_policies/generate.go

```diff
@@ -1,55 +1,53 @@
 package main
 
 import (
-	"fmt"
-	"bytes"
-	"math/rand"
+	// "fmt"
+	// "bytes"
+	// "math/rand"
+	authzpb "istio.io/api/security/v1beta1"
 )
 
 type generator interface {
-    generate(kind string, num int) (string, error)
+    generate(kind string, num int) (*authzpb.Rule, error)
 }
 
 type operationGenerator struct {
 }
 
-func (operationGenerator) generate(kind string, num int) (string, error) {
-	return "", fmt.Errorf("unimplimented")
+func (operationGenerator) generate(kind string, num int) (*authzpb.Rule, error) {
+	// TODO implement
+	condition := &authzpb.Rule{}
+    return condition, nil
 }
 
 type conditionGenerator struct {
 }
 
-func (conditionGenerator) generate(kind string, num int) (string, error) {
-	rule := bytes.Buffer{}
-	rule.WriteString("  - key: \n")
-
-    for i := 0; i < num; i++ {
-		rule.WriteString("    value")
+func (conditionGenerator) generate(kind string, num int) (*authzpb.Rule, error) {
```

Comment:
Currently only created very simple when with a key "request.headers[x-token]" and notValue of ["admin", "guest"]. Will reiterate with this